src/wobbly-effect.cpp

```diff
@@ -327,6 +327,33 @@ endless_shell_fx_wobbly_move_by (EndlessShellFXWobbly *effect,
     }
 }
 
+#define PAINT_BOX_OFFSET 1
+
+static gboolean
+endless_shell_fx_get_untransformed_paint_box (ClutterActor    *actor,
+                                              ClutterActorBox *box)
+{
+  /* Get the actor's parent class paint volume, bypassing affine
```

Comment:
This comment confuses me... There does not seem to be any involvement of the parent class.